src/dragonball/Cargo.toml

```
@@ -48,6 +49,8 @@ virtio-queue = { version = "0.7.0", optional = true }
 vm-memory = { version = "0.10.0", features = ["backend-mmap"] }
 crossbeam-channel = "0.5.6"
 fuse-backend-rs = "0.10.5"
+vfio-bindings = { version = "0.3.0", optional = true}
+vfio-ioctls =  { version = "0.1.0", optional = true}
```

Missing a space after `true`.

```suggestion
vfio-bindings = { version = "0.3.0", optional = true }
vfio-ioctls =  { version = "0.1.0", optional = true }
```

---

fixed